src/forms/Form/Form.tsx

```diff
@@ -30,6 +30,17 @@ const Form = (props: FormProps) => {
 
 	const { view } = useWindowResizer(type);
 
+	useEffect(() => {
+		const registerFields = async () => {

```

Comment:
The only issue I might see with this piece of code is the something similar to what's mentioned in https://simpleviewtools.atlassian.net/jira/software/c/projects/MOS/boards/82?modal=detail&selectedIssue=MOS-716 . Although they're not the same issue I feel it might also cause some memory leaks when unmounting and the async function still trying to be solved. I'd recommend adding something similar to the solutions in that ticket.